CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required (VERSION 3.8)
 ## CUSTOMISE
 
 # Define the application name and version.
-project (fastq_to_fasta VERSION 1.0.0)
+project (app-template VERSION 1.0.0)
```

Comment:
I'm assuming that `executable-name` == `repository-name`

Reply:
This shows that `fastq_to_fasta` should be its own example repository.